CHANGELOG.md

```diff
@@ -4,24 +4,47 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
-## [v2.1.0](https://github.com/voxpupuli/puppet-openvmtools/tree/v2.1.0) (2021-07-25)
+## [v3.0.0](https://github.com/voxpupuli/puppet-openvmtools/tree/v3.0.0) (2022-08-04)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-openvmtools/compare/v2.0.1...v2.1.0)
+[Full Changelog](https://github.com/voxpupuli/puppet-openvmtools/compare/v2.1.0...v3.0.0)
 
 **Breaking changes:**
 
-- Require puppet-epel over stahnma-epel [\#29](https://github.com/voxpupuli/puppet-openvmtools/pull/29) ([dhoppe](https://github.com/dhoppe))
+- Update from voxpupuli modulesync\_config; Drop RHEL 6; Drop Puppet 5 [\#53](https://github.com/voxpupuli/puppet-openvmtools/pull/53) ([TheMeier](https://github.com/TheMeier))
+
+**Closed issues:**
+
+- Add RockyLinux [\#50](https://github.com/voxpupuli/puppet-openvmtools/issues/50)
```

Comment:
I added the skip-changelog label here, please regenerate

Reply:
done